src/main/java/org/apache/cassandra/distributed/api/NodeToolResult.java

```diff
@@ -65,6 +74,16 @@ public Throwable getError()
         return error;
     }
 
+    public String getStdOut()
```

Comment:
for consistency, can we use `getStdout` or update asserts to be camel cased with the O?